### PR TITLE
Added database schema

### PR DESCRIPTION
**Pull request by rghosh8**
Please review my database schema. 

I named it **ads** with two tables in it: **similar_listing** and **things_to_do**.

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+);
+
+		
+CREATE TABLE rooms (
+  id INTEGER AUTO_INCREMENT PRIMARY KEY,
+  roomname CHAR(50)
+);
+
+
+ALTER TABLE listing ADD FOREIGN KEY (offering) REFERENCES offering (id);
```
Comment (timluu-zz) [inline schema.sql]: Where are your "listing" and "event" table referenced?

Comment (rghosh8) [reply schema.sql]: Thanks for the comments. I have updated the table. There was some issue with the git branching.

Comment (rghosh8) [reply schema.sql]: Please review.

Comment (ecuyle) [reply schema.sql]: How are you tying the `similar_listings` and `things_to_do` with a given offering id? I would expect to see an `offerings` table along with an `offerings_id` as a foreign key in both of your tables.

**Review by ecuyle — CHANGES_REQUESTED** [review]

**Review by ecuyle — APPROVED** [review]
LGTM. Good work!

**Review by timluu-zz — APPROVED** [review]
LGTM

**Pull request merged**